src/js/widgets/recommended/index.js

```diff
@@ -20,19 +18,11 @@ function widgetLoad( {
 	jQuery,
 	widgetId,
 } ) {
-	let uuid = false;
-	const cookieVal = getCookieValue( VISITOR_COOKIE_KEY_NAME );
-
-	if ( cookieVal ) {
-		try {
-			uuid = JSON.parse( unescape( cookieVal ) ).id;
-		} catch ( e ) {}
-	}
-
 	let fullUrl = apiUrl;
+	const uuid = personalized ? getUuidFromVisitorCookie() : undefined;
 
-	if ( personalized && uuid ) {
-		fullUrl += `&uuid=${ uuid }`;
+	if ( uuid ) {
```

Comment:
This changes is so we don't waste the i/o looking at cookies if there's no chance of using it. `uuid` is only referenced when `personalized` is truthy.